apps/test/unit/code-studio/pd/workshop_dashboard/components/workshop_managementTest.js

```diff
@@ -0,0 +1,123 @@
+import WorkshopManagement from '@cdo/apps/code-studio/pd/workshop_dashboard/components/workshop_management';
+import React from 'react';
+import {expect} from 'chai';
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+// import {Router} from 'react-router';
+
+describe("WorkshopManagement", () => {
+
+  const fakeRouter = {
+    createHref() {}
+  };
+  const context = {
+    router: fakeRouter
+  };
+
+  it("Renders expected buttons for not-started workshop", () => {
+    const mock = sinon.mock(fakeRouter);
+
+    mock.expects("createHref").withExactArgs("viewUrl").returns("viewHref");
+    mock.expects("createHref").withExactArgs("editUrl").returns("editHref");
+
+    const workshopManagement = shallow(
+      <WorkshopManagement
+        workshopId = {1}
+        viewUrl = "viewUrl"
+        editUrl = "editUrl"
+        onDelete={() => {}}
```

Comment:
We should probably also validate that the supplied `onDelete` function is wired up to the rendered delete button. Pass a stub function here, and then validate below something like 
```
const deleteButton = renderedButtons.get(2).props.children.first();
expect(deleteButton.props.onClick).to.eql(handleDeleteStub)
```